src/sentry/api/serializers/models/event.py

```diff
@@ -119,11 +134,21 @@ def get_attrs(self, item_list, user, is_public=False):
 
             (entries, entries_meta) = self._get_entries(item, user, is_public=is_public)
 
+            minidump = minidumps.get(item.event_id)
+
             results[item] = {
                 'entries': entries,
                 'user': user_data,
                 'contexts': contexts_data or {},
                 'sdk': sdk_data,
+                'minidump': {
```

Comment:
Should we generalize this to something like `crashFile` now, in case we ever add support for core dumps or other native crash formats?

Reply:
We use minidump in the documentation, having the same vocabulary everywhere makes life easier for customers, support and us.

Reply:
I'm specifically refering to the key used in the API for forward compatibility, nothing user visible at the moment. The alternative would be to add another key like this if we ever support another native crash report format.

In a similar fashion the project setting is called _"Store Native Crash Reports"_ and will include unreal crash reports (which wrap a minidump and contain more information).

Reply:
Unreal will store the minidump as attachment not the raw archive. Nobody can parsecthat one.

Reply:
Changed to crashFile internally and minidump externally for the specific type of crashfile.